controller/area_blackbox_test.go

```diff
@@ -59,190 +57,190 @@ func (rest *TestAreaREST) UnSecuredController() (*goa.Service, *AreaController)
+			// then
+			assert.Equal(t, *ca.Data.Attributes.Name, *created.Data.Attributes.Name)
+			fmt.Println(*created.Data.Relationships.Parent.Data.ID)
+			assert.Equal(t, parentID.String(), *created.Data.Relationships.Parent.Data.ID)
+
+			// try creating child area with different identity: should fail
+			otherIdentity := &account.Identity{
+				Username:     "non-space-owner-identity",
+				ProviderType: account.KeycloakIDP,
+			}
+			errInCreateOther := rest.db.Identities().Create(context.Background(), otherIdentity)
+			require.NoError(t, errInCreateOther)
+			svc, ctrl = rest.SecuredControllerWithIdentity(otherIdentity)
+			test.CreateChildAreaForbidden(t, svc.Context, svc, ctrl, parentID.String(), ca)
+		})
+
+		t.Run("Multiple Children", func(t *testing.T) {
+			/*
+				TestAreaREST ---> TestSuccessCreateMultiChildArea-0 ----> TestSuccessCreateMultiChildArea-0-0
+			*/
+			// given
+			fxt := tf.NewTestFixture(t, rest.DB, tf.Spaces(1), tf.Areas(1))
+			sp := fxt.Spaces[0]
+			parentArea := fxt.Areas[0]
+			parentID := parentArea.ID
+			ca := newCreateChildAreaPayload("TestSuccessCreateMultiChildArea-0")
+			owner, err := rest.db.Identities().Load(context.Background(), sp.OwnerID)
+			require.NoError(t, err)
+			svc, ctrl := rest.SecuredControllerWithIdentity(owner)
+			// when
+			_, created := test.CreateChildAreaCreated(t, svc.Context, svc, ctrl, parentID.String(), ca)
+			// then
+			assert.Equal(t, *ca.Data.Attributes.Name, *created.Data.Attributes.Name)
+			assert.Equal(t, parentID.String(), *created.Data.Relationships.Parent.Data.ID)
+			// Create a child of the child created above.
+			ca = newCreateChildAreaPayload("TestSuccessCreateMultiChildArea-0-0")
+			newParentID := *created.Data.Relationships.Parent.Data.ID
+			// when
+			_, created = test.CreateChildAreaCreated(t, svc.Context, svc, ctrl, newParentID, ca)
+			// then
+			assert.Equal(t, *ca.Data.Attributes.Name, *created.Data.Attributes.Name)
+			assert.NotNil(t, *created.Data.Attributes.CreatedAt)
+			assert.NotNil(t, *created.Data.Attributes.Version)
+			assert.Equal(t, newParentID, *created.Data.Relationships.Parent.Data.ID)
+			assert.Contains(t, *created.Data.Relationships.Children.Links.Self, "children")
+		})
+	})
 
-func (rest *TestAreaREST) TestShowAreaOKUsingExpiredIfModifedSinceHeader() {
-	// given
-	_, a := createSpaceAndArea(rest.T(), rest.db)
-	svc, ctrl := rest.SecuredController()
-	// when
-	ifModifiedSince := app.ToHTTPTime(a.UpdatedAt.Add(-1 * time.Hour))
-	res, _ := test.ShowAreaOK(rest.T(), svc.Context, svc, ctrl, a.ID.String(), &ifModifiedSince, nil)
-	//then
-	assertResponseHeaders(rest.T(), res)
+	rest.T().Run("Failure", func(t *testing.T) {
+		// given
+		fxt := tf.NewTestFixture(t, rest.DB, tf.Spaces(1), tf.Areas(1))
+		sp := fxt.Spaces[0]
+		parentArea := fxt.Areas[0]
+		parentID := parentArea.ID
+		childAreaPayload := newCreateChildAreaPayload(uuid.NewV4().String())
+		owner, err := rest.db.Identities().Load(context.Background(), sp.OwnerID)
```

Comment:
You already have access to the used identity by saying `owner := fxt.Identities[0]`. No need to load it explicitly. That is the beauty of the test fixture :)

Reply:
I'm in love with the test fixtures ;)